assets/report_template.Rmd

```diff
@@ -0,0 +1,1445 @@
+is a software for predicting functional abundances based only on marker gene sequences.
+Enzyme Classification numbers (EC), KEGG orthologs (KO) and MetaCyc ontology predictions were made for each sample.
+In folder [PICRUSt2/](../PICRUSt2/) are predicted quantifications for Enzyme Classification numbers (EC), see
+`EC_pred_metagenome_unstrat_descrip.tsv`, KEGG orthologs (KO), see `KO_pred_metagenome_unstrat_descrip.tsv`, MetaCyc ontology,
+see `METACYC_path_abun_unstrat_descrip.tsv`. Quantifications are not normalized yet, they can be normalized e.g. by the total sum per sample.
+"))
+```
+
+<!-- Section on methods -->
+
+# Methods
+
+```{r, results='asis'}
+if ( !isFALSE(params$dada2_ref_tax_title) ) {
+    cat("Taxonomic classification by DADA2:\n\n",
+        "- database: `", params$dada2_ref_tax_title, "`\n\n",
+        "- files: `", params$dada2_ref_tax_file, "`\n\n",
+        "- citation: `", params$dada2_ref_tax_citation, "`\n\n", sep = "")
+}
+
+if ( !isFALSE(params$qiime2_ref_tax_title) ) {
+    cat("Taxonomic classification by QIIME2:\n\n",
+        "- database: `", params$qiime2_ref_tax_title, "`\n\n",
+        "- files: `", params$qiime2_ref_tax_file, "`\n\n",
+        "- citation: `", params$qiime2_ref_tax_citation, "`\n\n", sep = "")
+}
+
+if ( !isFALSE(params$sintax_ref_tax_title) ) {
+    cat("Taxonomic classification by SINTAX:\n\n",
+        "- database: `", params$sintax_ref_tax_title, "`\n\n",
+        "- files: `", params$sintax_ref_tax_file, "`\n\n",
+        "- citation: `", params$sintax_ref_tax_citation, "`\n\n", sep = "")
+}
+
+if ( !isFALSE(params$mqc_plot) ) {
+    # with MultiQC
+    cat("[MultiQC](https://multiqc.info/) summarized computational methods in [multiqc/multiqc_report.html](../multiqc/multiqc_report.html).
+        The proposed short methods description can be found in [MultiQC's Methods Description](../multiqc/multiqc_report.html#nf-core-ampliseq-methods-description),
+        versions of software collected at runtime in [MultiQC's Software Versions](../multiqc/multiqc_report.html#software_versions),
+        and a summary of non-default parameter in [MultiQC's Workflow Summary](../multiqc/multiqc_report.html#nf-core-ampliseq-summary).\n\n")
+}
+# with & without MultiQC
+cat(paste0("
+Technical information to the pipeline run are collected in folder [pipeline_info](../pipeline_info),
+including software versions collected at runtime in file `software_versions.yml` (can be viewed with a text editor),
+execution report in file `execution_report_{date}_{time}.html`,
+execution trace in file `execution_trace_{date}_{time}.txt`,
+execution timeline in file `execution_timelime_{date}_{time}.html`, and
+pipeline direct acyclic graph (DAG) in file `pipeline_dag_{date}_{time}.html`.
+"))
+```
+
+<!-- Section on final notes -->
+
+# Final notes
+
+This report (file `summary_report.html`) is located in folder [summary_report](.) of the original pipeline results folder.
```

Comment:
Hm, that name (summary_report.html) is hardcoded, that should be fine, right?

Reply:
Yes I think so!